packages/sdk/src/sdk/services/Storage/Storage.ts

```diff
@@ -53,22 +53,24 @@ export class Storage implements StorageService {
   async editFile({
     uploadId,
     data,
-    auth
+    auth,
+    onProgress
   }: {
     uploadId: string
     data: { [key: string]: string }
     auth: AuthService
+    onProgress?: ProgressHandler
   }) {
     // Generate signature
-
     const signatureData = {
       upload_id: uploadId,
       timestamp: Date.now()
     }
-    const sigJson = JSON.stringify(sortObjectKeys(signatureData))
-    const signature = await auth.hashAndSign(sigJson)
+    const signature = await auth.hashAndSign(
```

Comment:
note to self - let you merge this first and I'll update this to `audiusWalletClient` in my branch